### PR TITLE
docs: clarify debug environment variables

### PR DESCRIPTION
**Pull request by vaivanov**

### CODE REVIEW

File: docs/environment-variables.md [hunk 1]
```diff
@@ -75,15 +75,11 @@ Warning: Enabling tracing will likely affect performance.
 
 ## `IPFS_FUSE_DEBUG`
 
-Enables fuse debug logging.
-
-Default: false
+Enables fuse debug logging. By default, fuse debug logging is disabled. Any value (even "") activates it.
```
Comment (Jorropo) [inline docs/environment-variables.md:78]:
```suggestion
If SET, enables fuse debug logging.

Default: false
```
The `Default:` is a convention most fields share, let's keep it.

File: docs/environment-variables.md [hunk 2]
```diff
@@ -75,15 +75,11 @@ Warning: Enabling tracing will likely affect performance.
...
 
 ## `YAMUX_DEBUG`
 
-Enables debug logging for the yamux stream muxer.
-
-Default: false
+Enables debug logging for the yamux stream muxer. By default, yamux debug logging is disabled. Any value (even "") activates it.
```
Comment (Jorropo) [inline docs/environment-variables.md:82]:
```suggestion
If SET, enables debug logging for the yamux stream muxer.

Default: false
```

**Review by Jorropo — CHANGES_REQUESTED** [review]
Thx, it's true that `Any value (even "") activates it.` is confusing.
I think it's better (goes straight to the point) that way.

**Comment by vaivanov** [comment]
Hi! Sounds great, thanks

**Comment by Jorropo** [comment]
CI is broken again, I'll merge anyway because it only touch markdown documentation.

**Pull request merged**